src/main/client/src/dashBoard/bookList/BookList.js

```diff
@@ -42,9 +42,9 @@ export class BookList extends Component {
                 key={book.id}
                 book={book}
                 reserveBook={this.props.reserveBook}
-                reservePopText={this.props.reservePopText}
+                popupText={book.popupText}
```

Comment:
if we're already passing in book then do we need to pass it in again, just being more specific?

Reply:
I'm not sure I understand the question?

Reply:
Oh wait I getcha